.configure-files/wordpress-sentry.properties.enc

```diff
@@ -0,0 +1,5 @@
+�7�"���wy�ɏ����1G�.Ѳ�z����.��p��P�����
+\�p�u�f�9ں�lX��sꞹ��
���S�5O����A���KԀ<ѷ)j4�:
+Iޏ�ρ��2�((L��X�H�i-)g_0�3�ߘsUKw��:���U�Ƞ
+�f�㗥�"��2�^��M�a�
+1pl�j����{s��2�
l�GU罈�X�v�C��&΢"*::6I^��g��W�\i2t�!�����r���9�-9�cq���_-P�w�����c�v�P4�0�Fu��o�H�T�1d*wM��c�-�L���Q
```

Comment:
No idea why GitHub doesn't consider this file as a binary, especially given we do have a line `.configure-files/*.enc binary` in our `.gitattributes` 🤔 🤷